helm/prometheus-operator-app/templates/prometheus-operator/deployment.yaml

```diff
@@ -62,28 +62,28 @@ spec:
             {{- end }}
             - --namespaces={{ $ns | join "," }}
             {{- end }}
-            - --logtostderr=true
             - --localhost=127.0.0.1
+            {{- if .Values.prometheusOperator.prometheusDefaultBaseImage }}
             {{- if (.Values.Installation) }}
-            - --prometheus-config-reloader={{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.prometheusOperator.prometheusConfigReloaderImage.repository }}:{{ .Values.prometheusOperator.prometheusConfigReloaderImage.tag }}
+            - --prometheus-default-base-image={{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.prometheusOperator.prometheusDefaultBaseImage }}
```

Comment:
Once config controller is used for prometheus operator, we will be able to remove this part and just define the prometheusDefaultBaseImage property in the helm chart, same for the alertmanager

Reply:
Ah thanks for this insight!

Reply:
Can you check a few lines below ? https://github.com/giantswarm/prometheus-operator-app/pull/79/files/dbb6746610a992def63e1234ddf1575ff7dab3dd#diff-e27896bc0794dd5a2ff25c437b60bb597c4b52f02e224d83e61d1bacb7d1f6f3R81